File: C#/DTO/Trip.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO
{
    public class Trip
    {
        public short TripId { get; set; }
        public string? Target { get; set; }
        public short? TypeId { get; set; }
        public string? TypeName { get; set; } = null;
        public DateTime? DateTrip { get; set; }
        public TimeSpan? LeavingTime { get; set; }
        public short? Duration { get; set; }
        public int? SeatsAvailable { get; set; }
        public int? Price { get; set; }
        public string? Image { get; set; }
        public bool NeedMedic { get; set; } 
    }
}

File: Angular/src/app/class/Trip.ts
import { Time } from "@angular/common";

export class Trips{
    slice(arg0: number, arg1: number): Trips {
      throw new Error('Method not implemented.');
    }
    constructor(
     
        public  tripId?: number,
        public  needMedic?: boolean,
        public  target?: string,
        public  typeId?: number,
        public  typeName?: number,  
        public  dateTrip?: Date, 
        public  leavingTime?: Time,
        public  duration?: number,
        public  seatsAvailable?:number,
        public  image?: string,
        public  price?: number,

        )
        {}
  }
  

